script/detect-sudo/customize.py

```diff
@@ -102,7 +104,7 @@ def prompt_sudo():
                 print(r.decode('utf-8'))  # Decode bytes to string
                 return 0
             except subprocess.TimeoutExpired:
-                reset_terminal()  # Reset terminal to sane state
+                reset_terminal()  # Reset terminal to same state
```

Comment:
sane is correct :)

Reply:
corrected in [31723a8](https://github.com/mlcommons/cm4mlops/pull/418/commits/31723a83a049bee4bee1455e1fba02720c9276e6)